src/index.d.ts

```diff
@@ -0,0 +1,54 @@
+declare interface ItkImage {
```

Comment:
Can we re-use the type definition in the `itk-wasm` package generated from:

 https://github.com/InsightSoftwareConsortium/itk-wasm/blob/master/src/core/Image.ts

?

Reply:
Yes!